apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -329,7 +320,7 @@ export default function SectionsSetUpContainer({
           onClick={e => {
             e.preventDefault();
             setIsSaveInProgress(true);
-            saveSection(sections[0]);
+            saveSection(sections[0], false);
```

Comment:
Here's what I've came up when trying to wrap this callback to useCallback hook. (https://github.com/code-dot-org/code-dot-org/pull/52058)
It's still not improving the number of renders though (due to current architecture of the logic)